index.html

```diff
@@ -220,6 +220,32 @@ <h3 class="small__resume_title">Skills</h3>
         </section>
 
         <!-- Contact Form -->
+        <section class="contacts">
+            <h2 class="contact_heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xvoyryqo" class="contact_form" method="post">
+                <ul class="form_items">
+                    <li class="form_list">
+                        <input type="text" class="name_input" placeholder="Fullname" maxlength="30">
+                    </li>
+                    <li class="form_list">
+                        <input type="email" name="user_email" class="email_input" placeholder="Email">
+                    </li>
+                    <li class="form_list">
+                        <textarea cols="30" rows="10" class="message_input" placeholder="Your Message"></textarea>
+                    </li>
+                    <li class="form_buttons">
+                        <button type="submit" class="form_submit">Get in Touch</button>
+                    </li>
+                </ul>
```

Comment:
**_this comment is optional but highly recommended_** 
- [ ] [OPTIONAL]  since the previous reviewer forgot to mention this, kindly add a `name="..."` attribute for both `textarea` and the `name input`, otherwise those data won't be submitted to formspree and you will not be able to know the name or the message of the sender